index.d.ts

```diff
@@ -1761,6 +1845,8 @@ declare namespace Eris {
       messageID: string,
       options: MessageWebhookContent
     ): Promise<Message<GuildTextableChannel>>;
+    emit<K extends keyof ClientEvents>(event: K, ...args: ClientEvents[K]): boolean;
+    emit(event: string, ...args: any[]): boolean;
```

Comment:
Event stuff should go at the bottom, if linter doesn't allow it, the config should be changed instead

Reply:
linter moves it around, alphabetical ordering